modules/claims_api/app/services/claims_api/disability_compensation/pdf_generation_service.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'claims_api/v2/disability_compensation_pdf_mapper'
+require 'pdf_generator_service/pdf_client'
+
+module ClaimsApi
+  module DisabilityCompensation
+    class PdfGenerationService < ClaimsApi::Service
+      EVSS_DOCUMENT_TYPE = 'L023'
+
+      def generate(claim_id, middle_initial) # rubocop:disable Metrics/MethodLength
+        log_service_progress(claim_id, 'pdf',
+                             "526EZ PDF generator started for claim #{claim_id}")
+
+        auto_claim = get_claim(claim_id)
```

Comment:
also, for these services does it make sense to change the `claim_id` to the `auto_claim` and just remove this first log and auto claim lookup? just need to change some logging from `claim_id` to `auto_claim.id` - also probably just tech debt / cleanup